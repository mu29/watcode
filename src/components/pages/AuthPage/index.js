import React from 'react'
import styled from 'styled-components'
import { Tab, TabPage, Header, GenericTemplate } from 'components'
import { SignInForm } from 'containers'
import { palette } from 'services/style'

const StyledTab = styled(Tab)`
  margin: 1rem 0;
  border: 0.0625rem solid ${palette('gray.40')};
  background-color: ${palette('white.default')};

  @media(max-width: 768px) {
    margin: 0;
    border: none;
  }
`

const AuthPage = () => (
  <GenericTemplate header={ <Header location="/auth" /> }>
    <StyledTab>
      <TabPage name="로그인">
        <SignInForm />
      </TabPage>
      <TabPage name="회원가입">
        <SignInForm />
      </TabPage>
    </StyledTab>
  </GenericTemplate>
)

export default AuthPage
