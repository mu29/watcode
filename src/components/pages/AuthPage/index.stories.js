import React from 'react'
import { storiesOf } from '@storybook/react'
import AuthPage from '.'

storiesOf('Pages/AuthPage', module)
  .add('기본', () => (
    <AuthPage />
  ))
