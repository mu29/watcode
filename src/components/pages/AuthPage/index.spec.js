import React from 'react'
import AuthPage from '.'

const wrap = (props = {}) => shallow(<AuthPage { ...props } />)

describe('<AuthPage />', () => {
  it('정상적으로 렌더링된다', () => {
    const wrapper = wrap()
    expect(wrapper).toMatchSnapshot()
  })
})
