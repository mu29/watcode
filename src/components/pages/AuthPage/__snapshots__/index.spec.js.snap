// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthPage /> 정상적으로 렌더링된다 1`] = `
<GenericTemplate
  header={
    <Header
      location="/auth"
    />
  }
>
  <Profile />
</GenericTemplate>
`;
